### DISCUSSION TITLE
Animated gif/movie command

### DISCUSSION THREAD

**Discussion by nord73** [cameroncros/OctoPrint-DiscordRemote#217] [Ideas]
Have you seen the Telegram remote plugin's command /gif? Would this be possible to accomplish here as well?

A static image is good, but a short movie would be better for adhesion/stability validation. Not something you would need on every print, but it sure is good to have on some prints. Their description of the commands are:

`/gif - Send a gif create from 20 images. You'll have to install CpuLimit to be able to create gif so please for raspberry connect to your raspberry with putty (for exemple) and execute sudo apt-get install cpulimit`

There is also a /supergif command creating a gif from 60 images.

The actual gif code is here: https://github.com/fabianonline/OctoPrint-Telegram/blob/dddb992769f0bec6e430ea0840c8b44ea2b145bb/octoprint_telegram/__init__.py#L1803

**Comment by cameroncros** [comment]
Sure, this sounds like a good idea. If you want to implement it and create
a PR, I would be happy to merge it.

I dont plan on implementing anything more on this plugin myself though, so
if you want this feature, eithe you or someone else will have to implement
it.
